src/js/actions/AppsActions.js

```diff
@@ -92,21 +92,21 @@ var AppsActions = {
       headers: {
         "Content-Type": "application/json"
       },
-      url: config.apiURL + "v2/apps/" + appId + "/restart"
-    })
-    .success(function (app) {
-      AppDispatcher.dispatch({
-        actionType: AppsEvents.RESTART_APP,
-        data: app,
-        appId: appId
-      });
+      url: `${config.apiURL}v2/apps/${appId}/restart`
```

Comment:
This is by design given from the API and doesn't do any harm.
But I also find it a little bit confusing.